app/services/publisher/service_provisioner.rb

```diff
@@ -20,7 +20,7 @@ class ServiceProvisioner
     LIVE_PRODUCTION = 'live-production'.freeze
 
     def service_metadata
-      service.to_json
+      service.to_json.inspect
```

Comment:
do we want to keep the `inspect` in there?

Reply:
I think so, when passed into the deployment.yaml.erb file it doesn't escape properly without the inspect and the test fails :(